### ISSUE TITLE
Live Console

### ISSUE THREAD

**Issue by farwl** [Minehut/Meta#39]
### Feature request

**Platform**
 Website

**Describe the feature**

Minehut does not have a live console. You have to navigate to your latest.log file which can be very annoying.

**How the feature is useful**

This is incredibly useful and many other hosts have it.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by drtshock** [comment]
Yes definitely very valuable. We plan on adding this, we're just not sure how soon we'll be able to get to this.
